docs/modules/ROOT/pages/how-tos/k3s_maintenance.adoc

```diff
@@ -0,0 +1,36 @@
+= K3s Maintenance
+
+== Kubernetes Maintenance
+
+The Kubernetes version is bound to the K3s version, so by updating K3s you also update the Kubernetes version.
+The https://github.com/rancher/system-upgrade-controller[System Upgrade Controller] from Rancher is used to upgrade the K3s version.
+The System Upgrade Controller is installed using a https://github.com/projectsyn/component-system-upgrade-controller[Project Syn component].
+
+In order to manage the K3s version the following System Upgrade Controller plan is used (the plan is also installed using the Project Syn component):
+
+[source,yaml]
+----
+include::example$suc_plan_k3s.yaml[]
+----
+
+Check the https://github.com/k3s-io/k3s/releases[K3s Release] page and update the value `version` with the version you want to upgrade to. Example: `version: v1.20.4+k3s1`
+
+This will immediatly trigger an upgrade of K3s. The container image `docker.io/rancher/k3s-upgrade` will replace the existing K3s binary and restart K3s.
```

Comment:
```suggestion
Once the plan is updated on the cluster, it will immediately trigger an upgrade of K3s.
The container image `docker.io/rancher/k3s-upgrade` will replace the existing K3s binary and restart K3s.
```